app/.eslintrc.json

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "next/core-web-vitals"
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "nava",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
Can we also include [type information with linting](https://typescript-eslint.io/docs/linting/type-linting)?